Molly/README.md

```diff
@@ -0,0 +1,168 @@
+### Homework #1
+(Due Monday 7/20)
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+**a)** `"1" == 1` True. When we convert "1" from a string into a number, 1 and 1 are equal.
+
+**b)** `"1" === 1` False. "1" is a string and 1 is a number. Therefore the two sides are not equal.
+
+**c)** `x == 'x'` False. x is undefined, which is always false-y
+
```

Comment:
Correct, but in this case if x's value is anything (undefined or not) other than 'x', x=='x' will return false, because it is comparing x's value to string 'x'.